fetch-examples/src/test/scala/GithubExample.scala

```diff
@@ -1,278 +1,278 @@
-      }
-  }
-
-  def repoContributors[F[_]: ConcurrentEffect](repo: Repo): Fetch[F, List[Contributor]] =
-    Fetch(repo, Contributors.source)
-
-  case class Project(repo: Repo, contributors: List[Contributor], languages: List[Language])
-
-  def fetchProject[F[_]: ConcurrentEffect](repo: Repo): Fetch[F, Project] =
-    (repoContributors(repo), repoLanguages(repo)).mapN({ case (contribs, langs) =>
-      Project(repo = repo, contributors = contribs, languages = langs)
-    })
-
-  def fetchOrg[F[_]: ConcurrentEffect](org: String) =
-    for {
-      repos    <- orgRepos(org)
-      projects <- repos.traverse(fetchProject[F])
-    } yield projects
-
-  def fetchOrgStars[F[_]: ConcurrentEffect](org: String): Fetch[F, Int] =
-    fetchOrg(org).map(projects => projects.map(_.repo.stargazers_count).sum)
-
-  def fetchOrgContributors[F[_]: ConcurrentEffect](org: String): Fetch[F, Int] =
-    fetchOrg(org).map(projects => projects.map(_.contributors.toSet).fold(Set())(_ ++ _).size)
-
-  def fetchOrgLanguages[F[_]: ConcurrentEffect](org: String): Fetch[F, Int] =
-    fetchOrg(org).map(projects => projects.map(_.languages.toSet).fold(Set())(_ ++ _).size)
-
-  "We can fetch org repos" in {
-    val io = Fetch.runLog[IO](fetchOrg[IO]("47deg"))
-
-    val (log, result) = io.unsafeRunSync
-
-    log.rounds.size shouldEqual 2
-  }
-
-  // Github HTTP api
-
-  val GITHUB: Uri = Uri.unsafeFromString("https://api.github.com")
-
-  private def fetchCollectionRecursively[F[_], A](c: Client[F], req: Request[F])(implicit
-      CF: BracketThrow[F],
-      E: EntityDecoder[F, List[A]]
-  ): F[List[A]] = {
-    val REL_NEXT = "rel=\"next\"".r
-
-    def hasNext(res: Response[F]): Boolean =
-      res.headers
-        .get(CaseInsensitiveString("Link"))
-        .fold(false)({ h => REL_NEXT.findFirstIn(h.value).isDefined })
-
-    def getNextLink(raw: String): F[String] = {
-      REL_NEXT
-        .findFirstMatchIn(raw)
-        .fold(
-          CF.raiseError[String](new Exception("Couldn't find next link"))
-        ) { m =>
-          CF.pure(m.before.toString.split(",").last.trim.dropWhile(_ == '<').takeWhile(_ != '>'))
-        }
-    }
-
-    def getNext(res: Response[F]): F[Uri] =
-      res.headers
-        .get(CaseInsensitiveString("Link"))
-        .fold(CF.raiseError[Uri](new Exception("next not found")))(raw =>
-          getNextLink(raw.value).map(Uri.unsafeFromString(_))
-        )
-
-    for {
-      result <- c.run(req).use[F, List[A]] {
-        case Status.Ok(res) =>
-          if (hasNext(res)) {
-            for {
-              repos <- res.as[List[A]]
-              nxt   <- getNext(res)
-              newReq = req.withUri(nxt)
-              moreRepos <- fetchCollectionRecursively(c, newReq)
-            } yield repos ++ moreRepos
-          } else
-            res.as[List[A]]
-        case res =>
-          CF.raiseError(new Exception(s"Couldn't complete request, returned status: ${res.status}"))
-      }
-    } yield result
-  }
-
-}
+///*
```

Comment:
What's the plan for this file? Should we delete it instead of commenting it out?

Reply:
I have uncommented all examples and only cross-compile them for scala2 versions. I assume we can wait until all libraries we depend on in the examples are published for scala3 to cross-compile the examples to scala3 as well.